### PR TITLE
Fix launcher for Python < 2.7

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: launcher/src/main/scripts/bin/launcher.py [hunk 1]
```diff
@@ -190,10 +190,11 @@ def build_java_execution(options, daemon):
     if options.log_levels_set and not exists(options.log_levels):
         raise Exception('Log levels file is missing: %s' % options.log_levels)
 
-    try:
-        subprocess.check_output(['java', '-version'], stderr=subprocess.STDOUT)
-    except (OSError, subprocess.CalledProcessError):
-        raise Exception('Java is not installed')
+    with open(os.devnull, 'w') as devnull:
```
Comment (findepi) [inline launcher/src/main/scripts/bin/launcher.py]: Technically this part (and `stdout=devnull` below) seems unrelated, so this could be 2 commits.

Comment (electrum) [reply launcher/src/main/scripts/bin/launcher.py]: How is this unrelated? The old method returns the output. This one just passes it through, so we need to discard the output to have the same behavior.
